web/src/features/hfiCalculator/components/CalculatedPlanningAreaCells.tsx

```diff
@@ -29,6 +29,33 @@ const CalculatedPlanningAreaCells = (props: CalculatedCellsProps) => {
     daily.date.toUTC().toMillis()
   )
 
+  const calculateDailyPrepLevels = () => {
+    const prepLevels: (number | undefined)[] = []
+    range(NUM_WEEK_DAYS).map(day => {
+      const meanIntensityGroup = dailyMeanIntensityGroups[day]
+      prepLevels.push(calculatePrepLevel(meanIntensityGroup))
+    })
+    return prepLevels
+  }
+  const calculateMeanPrepLevel = (
+    rawMeanIntensityGroups: (number | undefined)[]
+  ): number | undefined => {
+    // for now, prep level calculation assumed a fixed Fire Starts value of 0-1
+    if (isUndefined(rawMeanIntensityGroups)) {
+      return undefined
+    } else {
+      const existingDailies: number[] = []
+      rawMeanIntensityGroups.forEach(daily => {
+        if (!isUndefined(daily)) {
+          existingDailies.push(Math.round(daily))
+        }
+      })
+      return Math.round(
+        existingDailies?.reduce((a, b) => a + b, 0) / existingDailies.length
```

Comment:
I hate that Typescript doesn't have a nicer way of summing the numbers in an array. :/

Reply:
Yeah, that's what I could find :/

Reply:
You can use `lodash`'s `sum`: https://lodash.com/docs/4.17.15#sum